src/types.rs

```diff
@@ -2,6 +2,8 @@ use std::ops::Add;
 use std::ops::Neg;
 use std::ops::Sub;
 
+pub static COORD_PRECISION: f64 = 1e-1; // 0.1m
```

Comment:
I think the relevant libraries are [JTS](http://www.vividsolutions.com/jts/JTSHome.htm) and/or [GEOS](https://trac.osgeo.org/geos). Geometries in both libraries have a `PrecisionModel` to specify the precision of coordinates. The default is full double precision.